### PR TITLE
max-min

### PR DESCRIPTION
**Pull request by chandanghosh21**

### CODE REVIEW

File: min-max/maxMin.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    var maxMin = option.toUpperCase();
+    switch(maxMin){
+        case 'MAX': 
+        value = getMax(arr);
+        break;
+        case 'MIN':
+        value = getMin(arr);
+        break;
+        default:
+        value = "Invalid option. Provider 'MIN' or 'MAX'";
```
Comment (kakanghosh) [inline min-max/maxMin.js:17]: Typo: 'provider' -> 'provide'

File: min-max/maxMin.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+var arr = [5,10,45,100,80,7,9,108,1,500];
+
+var value = getMaxOrMin(arr,'min');
+console.log(value);
+
+
+function getMaxOrMin(arr,option) {
+    var maxMin = option.toUpperCase();
```
Comment (kakanghosh) [inline min-max/maxMin.js:8]: You can remove this line, take option.toUpperCase() in switch

File: min-max/maxMin.js (hunk 1 above)
Comment (chandanghosh21) [reply min-max/maxMin.js:17]: We always work with provider that's why mistakenly.

File: min-max/maxMin.js (hunk 2 above)
Comment (chandanghosh21) [reply min-max/maxMin.js:8]: ok, got it.

**Pull request merged**